CHANGELOG.md

```
@@ -4,6 +4,18 @@ All notable changes to DotAIOS will be documented in this file.
 
 ## [Unreleased]
 
+### Deprecated
```

markdownlint-cli2 (0.23.2)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)